### PR TITLE
Create card

### PR DESCRIPTION
**Pull request by Anna-Kuzmenko**

### CODE REVIEW

File: src/components/Card/Card.scss [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    display: block;
+  }
+
+  &__price {
+    display: flex;
+    align-items: start;
+    margin-top: 8px;
+
+    &--fullPrice {
+      @include prise;
```
Comment (VenomMB) [inline src/components/Card/Card.scss:63]: price

File: src/components/Card/Card.scss [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+      display: block;
+    }
+
+    &--discountPrise {
```
Comment (VenomMB) [inline src/components/Card/Card.scss:68]: price

File: src/components/Card/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import { Device } from '../../types/Device';
+import './Card.scss';
+import { FavoriteIcon } from '../../icons/FavoriteIcon';
+import { FavoriteFullIcon } from '../../icons/FavouriteFullIcon';
+
+interface Props {
+  phone: Device;
+}
+
+export const Card: FC<Props> = ({ phone }) => {
```
Comment (VenomMB) [inline src/components/Card/Card.tsx:12]: use memo

File: src/components/Card/Card.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    capacity,
+    // color,
+    ram,
+    // year,
+    image,
+  } = phone;
+
+  const onHandleClick = () => {
+    setIsFavorite(!isFavorite);
+  };
```
Comment (VenomMB) [inline src/components/Card/Card.tsx:31]: useCallback

File: src/components/Card/Card.tsx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+      <div className="card__content">
+        <a href="/" className="card__link">
+          <div className="card__photo_container">
+            <img
+              src={image}
+              alt={phoneId}
+              className="card__image"
+            />
+          </div>
+        </a>
```
Comment (VenomMB) [inline src/components/Card/Card.tsx:44]: use NavLink

File: src/components/Card/Card.tsx [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+        <a href="/" className="card__phone_link">
+          {name}
+        </a>
```
Comment (VenomMB) [inline src/components/Card/Card.tsx:48]: the same

File: src/components/Card/Card.tsx [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+              {fullPrice}
+            </del>
+          </p>
+        </div>
+
+        <div className="card__separate-line" />
+
+        <div className="card__char">
+          <p className="card__char--category_name"> Screen </p>
+          <p>{screen}</p>
```
Comment (VenomMB) [inline src/components/Card/Card.tsx:67]:
```suggestion
          <p className="card__char--category_name"> Screen </p>
          
          <p>{screen}</p>
```

File: src/components/Card/Card.tsx [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+        </div>
+
+        <div className="card__char">
+          <p className="card__char--category_name"> Capacity </p>
+          <p>{capacity}</p>
+        </div>
+
+        <div className="card__char">
+          <p className="card__char--category_name"> RAM </p>
+          <p>{ram}</p>
```
Comment (VenomMB) [inline src/components/Card/Card.tsx:77]:
```suggestion
          <p className="card__char--category_name"> Capacity </p>
          
          <p>{capacity}</p>
        </div>

        <div className="card__char">
          <p className="card__char--category_name"> RAM </p>
          
          <p>{ram}</p>
```

**Review by VenomMB — CHANGES_REQUESTED** [review]
Also, I think you can split this file, into small components. For example, buttons

File: src/components/Card/Card.scss (hunk 1 above)
Comment (Anna-Kuzmenko) [reply src/components/Card/Card.scss:63]: :see_no_evil:

File: src/components/Card/Card.tsx [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+        <div className="card__char">
+          <p className="card__char--category_name"> RAM </p>
+
+          <p>{ram}</p>
+        </div>
+
+        <div className="card__buttons">
+          <AddToCartButton />
+
+          <FavoriteButton />
```
Comment (VenomMB) [inline src/components/Card/Card.tsx:81]: Please read about hoc and custom hooks, I think you can use one component button)

**Review by VenomMB — CHANGES_REQUESTED** [review]

**Review by VenomMB — APPROVED** [review]
Good job

**Pull request merged**